src/sections/2021/section-2.js

```diff
@@ -49,7 +49,7 @@ const Section2 = () => (
       <GridCell start="2" span="3" startMD="3" spanMD="6" startLG="6" spanLG="7" className="util-margin-bottom-1xl util-margin-bottom-2xl@md">
         <Figure count="2.2" caption="Responses: 171">
           <PieChart
-            title="Individual contributors usually suggest design systems"
+            title="Individual contributor(s) suggested starting a design system"
```

Comment:
Should we also remove the "An" from the label for the first item in the pie chart key? My guess would be yes so that they match. (line 54 here)

<img width="987" alt="Screen Shot 2021-07-07 at 4 03 09 PM" src="https://user-images.githubusercontent.com/19694054/124821533-d5a00d00-df3c-11eb-84a0-11ef2a3cf61b.png">